src/playwrightTest.ts

```diff
@@ -152,7 +159,7 @@ export class PlaywrightTest {
     if (mode === 'list')
       allArgs.push('--reporter', 'null');
     const childProcess = spawn(node, allArgs, {
-      cwd: configFolder,
+      cwd: config.workspaceFolder,
```

Comment:
What was the reason to set the `cwd` for the process to the config directory?

Reply:
We resolve everything relative to that directory, for example when nested folders resolve to different versions of Playwright.